www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -20,20 +20,18 @@
 include_once __DIR__ . "/../../class/centreonLog.class.php";
 $centreonLog = new CentreonLog();
 
-try {
-    $pearDB->query(
-        "UPDATE `contact` SET `contact_autologin_key` = NULL WHERE `contact_autologin_key` =''"
-    );
-} catch (\PDOException $e) {
-    $centreonLog->insertLog(
-        2,
-        "UPGRADE : 19.10.2 Unable to set default contact_autologin_key"
-    );
-}
+//error specific content
+$versionOfTheUpgrade = 'UPGRADE - 20.04.0-beta.1 : ';
+$errorMessage = '';
 
-
-// Move broker xml files to json format
+/**
+ * @internal : Queries needing exception management and rollback if failing
```

Comment:
not sure internal is useful here

Reply:
There's many comments. I wanted to distinguish functionnal and structural one.
Should I remove it ?